src/services/dexScaner/DexPoolScannerService.ts

```diff
@@ -0,0 +1,38 @@
+import {inject, injectable} from 'inversify';
+import NodeCache from 'node-cache';
+import {getModelForClass} from '@typegoose/typegoose';
+
+import {LocalAssetRepository} from '../../repositories/LocalAssetRepository.js';
+import {DexSymbol} from '../../models/DexSymbol.js';
+
+@injectable()
+export class DexPoolScannerService {
+  verifiedTokenCache: NodeCache;
+  @inject(LocalAssetRepository) localAssetRepository!: LocalAssetRepository;
+
+  constructor() {
+    this.verifiedTokenCache = new NodeCache({stdTTL: 600});
+  }
+
+  async upsert(props: {
```

Comment:
move this to `DexScannerRepository`. imo this is more like repository management than service.

Reply:
Done